lotto.py

```diff
@@ -104,31 +106,35 @@ def get_random_winning_number():
     return random_winning_number
 
 
-def get_winning_number():  # 이렇게 오류 만들어서 예외처리 하는 건 장기적으로 안 좋을까요?
+def get_winning_number():
+    # 이렇게 오류 만들어서 예외처리 하는 건 장기적으로 안 좋을까요?
+
     while True:
         try:
             printer_please_input_winner()
             inputted_line = sys.stdin.readline().rstrip()
             if inputted_line == "r" or inputted_line == "R":  # 당첨번호 자동추첨하기
                 return get_random_winning_number()
             winning_number = list(map(int, inputted_line.split(',')))
-            if len(winning_number) != 6:  
+            if len(winning_number) != 6:
```

Comment:
이 로직을 아래와 같이 method로 빼보는 것은 어떨까요??

```suggestion
            def validate_winning_lotto_length(winning_number):
                if len(winning_number) !=6:
                    raise NotSixError
```

Reply:
여기에 try~ except 가 없어도 상위블록으로 넘어가서 except 부분에서 걸러지게 되요!!